docker/requirements.txt

```diff
@@ -3,10 +3,13 @@ colorcet
 imageio
 imageio-ffmpeg
 ipywidgets<9.0.0
-jupyter-server-proxy
 jupyterlab<5.0.0
 meshio
+nest_asyncio
```

Comment:
do we still need it?

Reply:
Good catch! Thanks!

Reply:
Ah turns out we do still need this due to the following:

https://github.com/pyvista/pyvista/blob/9273ed67f6fa721cfb75492f3f0ef541009227c6/pyvista/trame/jupyter.py#L373C5-L382

Reply:
I thought @alesgenova tried and thought it was not needed anymore.

Reply:
@alesgenova's work from #5240 is currently still dependent on nest_asyncio. @alesgenova, any ideas on how to work around this?

Reply:
I see. Thanks for the clarification.